packages/amplify-category-geo/src/commands/geo/update.ts

```diff
@@ -0,0 +1,33 @@
+import { chooseServiceMessageUpdate } from '../../provider-utils/awscloudformation/utils/constants';
+import { category } from '../../constants';
+import { supportedServices } from '../../provider-utils/supportedServices';
+
+const subcommand = 'update';
+
+module.exports = {
+  name: subcommand,
+  alias: ['configure'],
+  run: async (context: any) => {
+    const { amplify } = context;
+    const servicesMetadata = supportedServices;
+    return amplify
+      .serviceSelectionPrompt(context, category, servicesMetadata, chooseServiceMessageUpdate)
+      .then((result: {service: string}) => {
+        const providerController = servicesMetadata[result.service].providerController;
```

Comment:
unless @attilah feels strongly otherwise, I would like to see us get rid of this provider pattern in the category plugins. It's a big YAGNI and introduces a log of unnecessary complexity and indirection. This also includes getting rid of the `provider-utils/awscloudformation` directory structure. Instead, group files based on functionality and import the function you need directly rather than going through the "supportedServices" thing

Reply:
I agree. The current plugins have a complicated way of defining provider controllers that is redundant if CFN is the only supported provider. Made some changes to make it simple.